README.md

```diff
@@ -28,40 +29,96 @@ While they are hierarchically related, the results of the first step are version
+
+For apptainer/singularity:
 
 ```console
 make analysis-singularity
 ```
+_or_
+
+For OCI containers you can run with docker (default) or podman by setting the environment variable `OCI_BINARY=podman`
 
 ```console
 make analysis-oci
 ```
 
-This produces a PDF article and its associated elements (mainly volumetric binary data, `.nii.gz` files) which will be stored in a datestamped and annotated directory under `outputs/`.
-A number of outputs are recorded via `git-annex` and therefore present in this repository, and your output can also be saved and recorded.
 
+### II. Reexecuting the Meta-Article
 
+To avoid confusion, we use the term 'article' to refer to a version of the OPFVTA article, and 'meta-article' to refer to the paper regarding the reexecution process and findings.
 
-### II. Reexecuting the Meta-Article
+Generation of the meta-article uses files generated by the OPFVTA analysis which are expected to be in the `outputs/` directory.
+Prior to generating the meta-article, `outputs/` must contain the data from previous analyses, which is not locally available by default.
 
-This uses the aforementioned PDF files in `outputs/` in order to generate dynamic graphical elements, and subsequently compiles them alongside the document text via LaTeX into a novel and fully distinct article PDF.
-To avoid confusion, please make sure you understand this is *not* another version of the OPFVTA article but a fully different text.
+Note: Regenerating the OPFVTA article will create an additional pdf, but the previous pdfs are required to compare.
 
-This phase requires fetching the actual binary content for the myriad PDF outputs of OPFVTA reexecution, and then running the `make article` target:
+To fetch the OPFVTA analysis outputs:
 
 ```console
 datalad get outputs/*/article.pdf
-make article
 ```
 
-A finer point here is that the dynamic elements of this article can be cached.
-If you are not merely trying to get a PDF to read or working on the human-readable text — but instead working on the figure-generating code — it is advisable to always clean the dynamic elements in between re-making the article via the dedicated target.
+Finally we generate new graphical elements and compile the text via LaTeX into a novel meta-article PDF.
+
+The meta-article can then be generated by a container with all of the dependencies preinstalled using:
+
+```console
+make container-article
+```
+
+_or_
+
+If you prefer to run the generation outside of a container, you will need to install dependencies (suggested to use distribution package manager, packages below are debian names):
+	- laTex
+	- biber
+	- datalad
+	- diff-pdf
+	- graphviz
+	- python3-matplotlib
+	- python3-pandas
+	- python3-seaborn
+	- python3-sklearn
+	- python3-statsmodels
+	- python3-yaml
+
+You will also need to install sourceserifpro font using the tlmgr.
```

Comment:
@asmacdo 

I have tried `make article-clean && make article-container` without the `tlmgr` code ( https://github.com/con/opfvta-replication-2023/commit/f1399d942501f2914f43d2218d636cca9b52a0f9 ) and it worked. Can you confirm that it also works for you without that? if so, we can drop this.

Reply:
see [other ](https://github.com/con/opfvta-replication-2023/pull/38#discussion_r1339183150)